exe/qt.conf

```diff
@@ -1,4 +0,0 @@
-# solves problem with binaries not launching on mac.
```

Comment:
Looks like this might still be used... although I thought I grepped for it. From actions:

>150595 INFO: Appending 'datas' from .spec
Unable to find "D:\a\invest\invest\exe\qt.conf" when adding binary and data files.

Reply:
Looks likes it's in `exe/invest.spec`

>     'datas': [('qt.conf', '.'), proj_datas],